test/kitchen/kitchen-azure-common.yml

```diff
@@ -41,6 +41,9 @@ driver:
   <% else %>
```

Comment:
Sanity check: Line 10 (I can't highlight it in Github) there is some references of `CI_PIPELINE_ID`

Reply:
The one on line 10 is fine (it was already like this in the old file: https://github.com/DataDog/datadog-agent/pull/4362/files#diff-5c7b52dfb2403a10d5cf19cf1b755deaL10-L14)